doc/api/tls.markdown

```diff
@@ -202,63 +202,64 @@ established it will be forwarded here.
 
 `function (sessionId, sessionData, callback) { }`
 
-Emitted on creation of TLS session. May be used to store sessions in external
+Emitted on creation of a TLS session. May be used to store sessions in external
 storage. `callback` must be invoked eventually, otherwise no data will be
-sent or received from secure connection.
+sent or received from the secure connection.
 
-NOTE: adding this event listener will have an effect only on connections
-established after addition of event listener.
+NOTE: adding this event listener will only have an effect on connections
```

Comment:
For consistency, and since this is a full sentence, we could use _"Adding"_ instead of _"adding"_.